deploy/ansible/mongodb.yml

```diff
@@ -5,17 +5,80 @@
       become: true
 
 - hosts: all
-  roles:
-    - undergreen.mongodb
-  vars:
-    mongodb_net_http_enabled: true
-    mongodb_net_bindip: 0.0.0.0
-    mongodb_pymongo_pip_version: 4.1.1
-    mongodb_security_authorization: "enabled"
-    mongodb_users:
-      - {
-          name: "{{ mongodb_user }}",
-          password: "{{ mongodb_pass }}",
-          roles: readWrite,
-          database: "{{ mongodb_db }}"
-        }
+  become: true
+  tasks:
+    - name: Install aptitude using apt
+      apt:
+        name: aptitude
+        state: latest
+        update_cache: yes
```

Comment:
Do we need this?

Reply:
Good call.  I was meant to test without is. I do not think we need it. Will confirm.

Reply:
Removed!